sdk/core/azure-core-http-jdk-httpclient/src/main/java/com/azure/core/http/jdk/httpclient/JdkHttpClientBuilder.java

```diff
@@ -227,6 +230,26 @@ public JdkHttpClientBuilder configuration(Configuration configuration) {
         return this;
     }
 
+    /**
+     * Sets the {@link HttpProtocolVersion protocol versions} that the {@link HttpClient} can use.
+     * <p>
+     * {@link HttpProtocolVersion#HTTP_1_1} must be included in the set of versions.
+     * <p>
+     * If the value is not set, only {@link HttpProtocolVersion#HTTP_1_1} can be used.
+     *
+     * @param protocolVersions The {@link HttpProtocolVersion protocol versions} that the {@link HttpClient} can use.
+     * @return The updated {@link JdkHttpClientBuilder} object.
+     */
+    public JdkHttpClientBuilder setProtocolVersions(EnumSet<HttpProtocolVersion> protocolVersions) {
```

Comment:
nit: We generally don't use `set` prefix on builder methods.

Reply:
Will fix this once we resolve the larger question of overall design.